content/en/universal_service_monitoring/_index.md

```diff
@@ -731,7 +731,10 @@ SERVICE=<service>
 
 **For services running on IIS:**
 
-1. Install the [Datadog Agent][1] (version 6.41 or 7.41 and later) with the network driver component enabled. During installation, pass `ADDLOCAL="MainApplication,NPM"` to the `msiexec` command, or select **Network Performance Monitoring** when running the Agent installation through the UI.
+1. Install the [Datadog Agent][1] (version 6.41 or 7.41 and later) with the network driver component enabled.
+
+   _(version 7.45+)_ During installation select the check box to consent to non-open source software.
```

Comment:
should we wait with the until 7.45 is finally released?

Reply:
Yes, definitley should wait, I was thinking we should have something ready vs. waiting till after it is released. Not sure how to time it, but maybe all approve and then when released we merge it the same day?